### DISCUSSION TITLE
Possible new features

### DISCUSSION THREAD

**Discussion by aritra-mukhopadhyay** [amepproject/amep#18] [Ideas]
- Finite size scaling analysis
- Van Hove functions (SVH) - may be useful for tracking anomalous fluctuations in clusters. See https://arxiv.org/pdf/2401.14745.pdf
- Minkowski functionals for clusters. See https://journals.aps.org/prresearch/abstract/10.1103/PhysRevResearch.4.L042046
- Entropy production
- Cluster growth rate by measuring the perimeter of structures (could be useful for connected clusters/structures/patterns in continuum simulations). See https://journals.aps.org/prb/abstract/10.1103/PhysRevB.39.11956
- Total pressure = Active pressure + Interaction pressure (https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.115.098301)

**Comment by hechtprojects** [comment]
For entropy production, see also https://github.com/amepproject/amep/issues/14. The proposed method there is a general approach that does not require any knowledge about the underlying equations of motion. 

@aritra-mukhopadhyay Do you have any other method in mind?
